relaxed.module

```diff
@@ -73,6 +73,26 @@ function relaxed_entity_access(EntityInterface $entity, $operation, AccountInter
 }
 
 /**
+ * Implements hook_entity_create_access().
+ */
+function multiversion_entity_create_access(AccountInterface $account, array $context, $entity_bundle) {
+  /** @var \Drupal\multiversion\MultiversionManagerInterface $manager */
+  $multiversion_manager = \Drupal::service('multiversion.manager');
+  $bundles_info = \Drupal::entityManager()->getAllBundleInfo();
+  foreach ($bundles_info as $entity_type_id => $bundles) {
+    if (in_array($entity_bundle, array_keys($bundles))){
+      $entity_type = \Drupal::entityManager()->getDefinition($entity_type_id);
+      if ($multiversion_manager->isSupportedEntityType($entity_type) || $entity_type_id == 'replication_log') {
+        return AccessResult::allowedIfHasPermission($account, 'perform content replication');
```

Comment:
@dickolsson This maybe is not specifically for attachment replication, it fixes an access problem I've detected when doing this replication.